src/libtomahawk-playdarapi/Api_v1_5.cpp

```diff
@@ -105,7 +105,7 @@ Api_v1_5::playback( QxtWebRequestEvent* event, const QString& command )
             } else {
                 trackInfo.insert( "resolvedBy", "<unknown resolver>" );
             }
-            trackInfo.insert( "score", currentTrack->score() );
+//FIXME?            trackInfo.insert( "score", currentTrack->score() );
```

Comment:
A resolver weighting is possible, but it has its own limitations: it's only accurate within the domain of one particular resolver. Comparing the weight of a Soundcloud result with a Spotify result would be meaningless, since the resolvers have no idea of each other's weighting and how they compare.

This makes me wonder how useful this weighting is in reality. Discuss.